.github/workflows/upload-to-s3.yaml

```diff
@@ -35,7 +37,7 @@ jobs:
       - name: Configure aws credentials
         uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
should this be node18? If that exists

Reply:
Nice catch! I missed this. Will look into this.

Reply:
Looks like there is currently not a node18 variant of this. And this is an interim measure until v2 is released.
https://github.com/aws-actions/configure-aws-credentials/issues/489#issuecomment-1278145876

For now, this is not affecting our builds as the runtime is not used for anything we create, as it's a action used for configuring/using aws credentials.

Reply:
did look a bit weird, thanks for checking it out! I've resolved this thread.